mappings/net/minecraft/entity/vehicle/impl/EntityBoat.mapping

```diff
@@ -17,22 +17,34 @@ CLASS none/abj net/minecraft/entity/vehicle/impl/EntityBoat
 		METHOD a getType (Ljava/lang/String;)Lnone/abj$b;
 			ARG 0 name
 		METHOD b getId ()I
+		METHOD values values ()[Lnone/abj$b;
 	FIELD d BOAT_TYPE Lnone/kj;
 	METHOD <init> (Lnone/aiw;DDD)V
 		ARG 0 world
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
+	METHOD a (DDDFFIZ)V
+		ARG 0 x
+		ARG 1 y
+		ARG 2 z
+		ARG 3 yaw
+		ARG 4 pitch
 	METHOD a deserializeCustomData (Lnone/dt;)V
 		ARG 0 tag
 	METHOD a damage (Lnone/rr;F)Z
 		ARG 0 source
 		ARG 1 damage
+	METHOD a (Lnone/sg;)V
+		ARG 0 entity
 	METHOD b interact (Lnone/aam;Lnone/rb;)Z
 		ARG 0 player
 		ARG 1 hand
 	METHOD b serializeCustomData (Lnone/dt;)V
 		ARG 0 tag
 	METHOD i initDataTracker ()V
+	METHOD j getBoatItem ()Lnone/aeq;
```

Comment:
My suggestion would be to use `asItemStack` as the standard method to return a new stack that represents the object in some way. (As opposed to, say, a backpack entity returning its internal stack which retains durability, enchantments and such.)